Makefile

```diff
@@ -78,6 +79,12 @@ test-rules:
 shellcheck:
 	${DOCKER_CLI} run -v "${PWD}:/mnt" koalaman/shellcheck:stable $(shell find . -type f -name "*.sh" -not -path "*vendor*")
 
+lint-markdown-format:
+	${DOCKER_CLI} run -v ${PWD}:/workdir davidanson/markdownlint-cli2:v0.9.2 --config .markdownlint-cli2.jsonc
```

Comment:
Can you move the version into a variable?

Reply:
That's true
I've defined the new variable of the version